config/common.yaml

```diff
@@ -482,14 +495,6 @@ servers:
                       value: "[[$.ServerPassword]]"
                     - name: DROOLS_SERVER_FILTER_CLASSES
                       value: "true"
-                    - name: KIE_SERVER_CONTROLLER_USER
-                      value: controllerUser
-                    - name: KIE_SERVER_CONTROLLER_PWD
-                      value: "[[$.ControllerPassword]]"
-                    - name: KIE_SERVER_CONTROLLER_SERVICE
-                      value: "[[$.ApplicationName]]-[[$.Console.Name]]"
-                    - name: KIE_SERVER_CONTROLLER_PROTOCOL
-                      value: ws
```

Comment:
Isn't OpenShift Startup Strategy optional? In other words, the user can still use the old startup strategy, no? In that case, these have to remain.

Reply:
https://github.com/rhtevan/kie-cloud-operator/blob/f49547408accbc1a28b880d7537618f31d64e137/config/common.yaml#L491

OSSS is set. Does KieApp CR allow user to change strategy?

Reply:
@bmozaffa Looks like the KieApp CR allows user to add arbitrary new env variables as well as overriding predefined ones. In such case, would it be cleaner if we remove those Controller related envs from the base yaml, but provide either example CR or new 'deploy/crs/.yaml' to add it back? Wdyt? For now, added those back.

Reply:
Hadn't you said that KIE servers still need to register with the embedded controller, for some other purposes?

Reply:
Yes. the lifecycle hook script from KIE Server Pod, connects to BC using KIE_ADMIN_USER (Not CONTROLLER_UER) for REST/API calls.

Reply:
oh right, sorry if I'm repeating the same discussion again, but... this is a new dependency and we previously did not need the admin user for anything system related. Can you use controller user, or are there access issues? Again, sorry if I'm failing to remember this from last time!

Reply:
Can we use Admin.User for now? 
Couple of reasons:
1. It is an existing user with elevated privileges configured at both Templates and Operator common.yaml right now.
2. ControllerUser/Pass has already been removed from KIE Server DC env section of all RHPAM and RHDM Templates.
3. Need a system account for connecting to BC/WB internal REST/API with sufficient privileges. At the moment Admin.User works.

Reply:
Yes, agree, we can for now. However, we should review this and make required improvements in 7.5. Please open a jira for yourself to this effect, targeting 7.5. Goal is to use a specific user for this purpose, most likely still called controllerUser, with only required permissions. We would then remove adminUser in SSO type of scenarios.

Reply:
Or something like 'WBServiceUser'.

Reply:
A follow up JIRA created (https://issues.jboss.org/projects/RHPAM/issues/RHPAM-2175)